modules/swagger-codegen/src/main/java/com/wordnik/swagger/codegen/languages/RetrofitClientCodegen.java

```diff
@@ -0,0 +1,190 @@
+package com.wordnik.swagger.codegen.languages;
+
+import com.wordnik.swagger.codegen.*;
+import com.wordnik.swagger.models.Operation;
+import com.wordnik.swagger.models.properties.*;
+
+import java.util.*;
+import java.io.File;
+
+public class RetrofitClientCodegen extends DefaultCodegen implements CodegenConfig {
```

Comment:
Then perhaps we should have an AbstractJavaCodegen that all these derive from or put that shared functionality in some kind of helper (i.e. inheritance vs composition). If you don't have time for either please let me know so I can open a corresponding ticket after your PR has been merged. Thank you!